[200~From 78a4103946854e4e526b0cf407f0b09610b40988 Mon Sep 17 00:00:00 2001
From: hscham <hscham@chromium.org>
Date: Mon, 1 Jun 2020 16:50:16 +0900
Subject: [PATCH] libchrome: Remove glib dependency.

BUG=chromium:361635
TEST=Ran Pre-CQ.

Change-Id: I8a583851d51c5eaa24bb1506d45d6566f760595c
---
 build/build_config.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/build/build_config.h b/build/build_config.h
index 0a0024088..b0e2aa26e 100644
--- a/build/build_config.h
+++ b/build/build_config.h
@@ -39,10 +39,6 @@
 #elif !defined(__ANDROID_HOST__)  // Chrome OS
 
 #define OS_CHROMEOS 1
-// TODO: Remove these once the GLib MessageLoopForUI isn't being used:
-// https://crbug.com/361635
-#define USE_GLIB 1
-#define USE_OZONE 1
 
 #endif  // defined(__ANDROID__)
 
-- 
2.27.0.rc2.251.g90737beb825-goog

